js/config.js

```diff
@@ -120,6 +120,32 @@ export const notify_in = _meta('notify-in');
  */
 export const is_territory_enabled = _jsonMeta('territory-enabled');
 
+/**
+ * Detect HiDPI screens
+ */
+export const hidpi = (window.devicePixelRatio > 1 || (
+    window.matchMedia
+    && window.matchMedia('(-webkit-min-device-pixel-ratio: 1.25),(min-resolution: 120dpi)').matches)
+);
+
+/**
+ * Attributions for map tiles
+ */
+export const tiles_attributions = '&copy;' + [
```

Comment:
Great, this way I'll be able to use it here: https://github.com/opendatateam/udata/pull/501/files#diff-4edd25845e3334ac0c75ecf4718311f3 :)